core/server/master/src/test/java/alluxio/master/block/BlockMasterTest.java

```diff
@@ -86,7 +86,7 @@ public void before() throws Exception {
     mClock = new ManualClock();
     mExecutorService =
         Executors.newFixedThreadPool(2, ThreadFactoryUtils.build("TestBlockMaster-%d", true));
-    mMaster = new BlockMaster(registry, factory, mClock,
+    mMaster = new BlockMasterFactory().create(registry, factory, mClock,
```

Comment:
for consistency with FileSystemMaster, use the DefaultBlockMaster constructor here instead